schemas/vnsutils.json

```diff
@@ -143,6 +143,14 @@
                 "sectionEnd": "Data Parameters",
                 "propertyOrder": 150
             },
+            "vcenter_vm_folder": {
+                "type": "string",
+                "title": "Specify a path where VM can be brought up",
+                "default": "/usr/bin/ovftool",
```

Comment:
This doesn't make sense as a default folder... Shouldn't this be empty?

Reply:
Yes, I am sorry about that, I initially wrote that, and then overlooked that part and forgot to remove it, true it should be empty now I made the changes, really sorry about that